tests/test_snapshot/test_metadata.py

```diff
@@ -3,17 +3,36 @@
 from typing import Dict, Type
 
 import pytest
+from numpy import array
```

Comment:
Ditto above - I always use 'import numpy as np' so I'm also curious if I should also only be importing the functions I need or if it's just personal preference

Reply:
unsure. @panzer what do you think?